skfda/exploratory/visualization/_boxplot.py

```diff
@@ -249,8 +242,19 @@ class Boxplot(FDataBoxplot):
 
     """
 
-    def __init__(self, fdatagrid, depth_method=ModifiedBandDepth(), prob=[0.5],
-                 factor=1.5):
+    def __init__(
+        self,
+        fdatagrid: FDataGrid,
+        depth_method = ModifiedBandDepth(),
```

Comment:
The object should be created inside the function body. Also, the type is missing.

Reply:
true, solved

Reply:
I think you misunderstood my comment. The only object that should be created inside is the default one (so, initialize it to `None` and check if it is `None` in the function body, and create it if that is the case). However, in function calls the object should still be passed.